### PR TITLE
Add VCR for caching external calls that keep getting blocked

### PR DESCRIPTION
**Pull request by kevinreedy**
@robbkidd thoughts?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,8 +3,11 @@ ruby '2.4.2'
 
 gem 'activesupport'
 gem 'flydays', git: 'https://github.com/kevinreedy/flydays.git', ref: '2e8d88f446bd5257c3e6da865d656f1f697dc4a0'
+gem 'rake'
 gem 'sinatra'
 gem 'shotgun'
+gem 'vcr'
+gem 'webmock'
```
Comment (robbkidd) [inline Gemfile:10]: I'd move these into a:

```
group :test do
  gem 'vcr'
  gem 'webmock'
end
```

... so the fakery gems aren't needed in production.

Comment (robbkidd) [reply Gemfile:10]: Er ... or, I seem to be misunderstanding their use. Imma read more code.

Comment (robbkidd) [reply Gemfile:10]: Ooooh. Uh. I see what you're doing here.

I'm not sure how I feel about it.

tenor-246286052

**Comment by robbkidd** [comment]
When/where have you seen which calls getting blocked?

**Comment by kevinreedy** [comment]
@robbkidd they're getting block on Heroku. Southwest returns a 403 most of the time.

**Pull request merged**